src/main/java/seedu/address/logic/commands/iteration/AddIterationCommand.java

```diff
@@ -62,8 +68,23 @@ public CommandResult execute(Model model) throws CommandException {
                     activeCommission.getTitle().toString()));
         }
 
-        activeCommission.addIteration(toAdd);
-        return new CommandResult(String.format(MESSAGE_ADD_ITERATION_SUCCESS, toAdd,
+        String src = toAdd.getImagePath().path;
+        String dst = "saved/" + activeCommission.getTitle() + activeCommission.getIterationCounter() + ".png";
```

Comment:
Since different commissions can have the same title for different customers, is there a better way of naming this image, perhaps using the commissions hash?